app/src/main/java/openfoodfacts/github/scrachx/openfood/models/Allergen.java

```diff
@@ -105,4 +123,8 @@ public String toString() {
                 ", idAllergen='" + idAllergen + '\'' +
                 '}';
     }
+
+    public String getEnable() {
```

Comment:
Mhhh after rebuilding the project, GreenDao add automatically this getter.... Even if I delete it...